docs/conf.py

```diff
@@ -36,10 +37,8 @@
               'sphinx.ext.viewcode',
               'jupyter_sphinx',
               'reno.sphinxext',
+              'qiskit_sphinx_theme',
              ]
-html_static_path = ['_static']
-templates_path = ['_templates']
```

Comment:
This needs to be set because you are using the `_templates` folder.

Reply:
Thanks, just re-added that.